docker-compose.yml

```diff
@@ -1,11 +1,37 @@
-app:
-  mem_limit: 300m
-  memswap_limit: 1g
-  env_file:
-    .env
-  build: .
-  ports:
-    - '3000:8000'
-  volumes:
-    - .:/home/nodejs/app
-    - /home/nodejs/app/node_modules
+version: '2'
+
+services:
+  mongodb:
+    container_name: ya-mongo-db
+    image: alexpunct/mongo:3.4
+    volumes:
+      - mongostorage:/data/db
+    ports:
+      - "${MONGODB_EXPOSED_PORT}:27017"
```

Comment:
So it should resolve to `27017:27017 `, note though: If you already have running mongo on your machine, you would have a conflict there

Reply:
yeah, thats true